src/pages/editor/ChannelClaimTableRow.tsx

```diff
@@ -41,67 +56,108 @@ export const ChannelClaimTableRow = ({ claimedChannel, channelType }: ChannelCla
+            }
+          />
+        </StyledTableCell>
+        <StyledTableCell sx={{ display: 'flex', flexWrap: 'wrap', gap: '0.25rem' }}>
+          {claimedChannel.channelClaim.constraint.scope.map((scope) => (
+            <Chip key={scope} variant="filled" color="primary" label={t(`registration.publication_types.${scope}`)} />
+          ))}
+        </StyledTableCell>
+        {isOnSettingsPage && (
+          <>
+            <StyledTableCell>
+              <Tooltip title={t('common.remove')}>
+                <IconButton
+                  data-testid={`delete-channel-claim-${channelIdentifier}`}
+                  onClick={() => setOpenConfirmDialog(true)}
+                  size="small"
+                  sx={{ width: '1.5rem', height: '1.5rem' }}>
+                  <CloseOutlinedIcon fontSize="inherit" />
+                </IconButton>
+              </Tooltip>
+            </StyledTableCell>
+          </>
         )}
-      </StyledTableCell>
-      <StyledTableCell aria-live="polite" aria-busy={organizationQuery.isPending}>
-        {organizationQuery.isPending ? (
-          <Skeleton width={300} />
-        ) : organizationName ? (
-          <Typography>{organizationName}</Typography>
-        ) : (
-          <Typography sx={{ fontStyle: 'italic' }}>{t('common.unknown')}</Typography>
-        )}
-      </StyledTableCell>
-      <StyledTableCell>
-        <StyledChip
-          variant="filled"
-          color="secondary"
-          size="small"
-          sx={{
-            bgcolor: publishingPolicy === 'Everyone' ? 'publishingRequest.main' : 'centralImport.main',
-          }}
-          label={t(`editor.institution.channel_claims.access_policies.${publishingPolicy}`)}
-          icon={
-            publishingPolicy === 'Everyone' ? <LockOpenIcon fontSize="small" /> : <LockOutlineIcon fontSize="small" />
-          }
-        />
-      </StyledTableCell>
-      <StyledTableCell>
-        <StyledChip
-          variant="filled"
-          color="secondary"
-          size="small"
-          sx={{
-            bgcolor: editingPolicy === 'Everyone' ? 'publishingRequest.main' : 'centralImport.main',
-          }}
-          label={t(`editor.institution.channel_claims.access_policies.${editingPolicy}`)}
-          icon={editingPolicy === 'Everyone' ? <LockOpenIcon fontSize="small" /> : <LockOutlineIcon fontSize="small" />}
+      </TableRow>
+
+      <ConfirmDialog
+        open={openConfirmDialog}
+        title={t('editor.institution.channel_claims.delete_channel_claim')}
+        isLoading={deleteMutation.isPending}
+        onAccept={async () => await deleteMutation.mutateAsync()}
+        onCancel={() => setOpenConfirmDialog(false)}>
+        <Trans
+          i18nKey="editor.institution.channel_claims.delete_channel_claim_description"
+          values={{ name: channelName }}
+          components={{ span: <span style={{ fontWeight: 'bold' }} /> }}
         />
```

Comment:
Teksten her burde sikkert hatt sin egen `<p>`? Nå ligger det bare en `<span>` inni der

![bilde](https://github.com/user-attachments/assets/738e59d4-3d76-4ee3-99c3-4ce3a7a53b55)

```suggestion
        <Typography>
          <Trans
            i18nKey="editor.institution.channel_claims.delete_channel_claim_description"
            values={{ name: channelName }}
            components={{ span: <span style={{ fontWeight: 'bold' }} /> }}
          />
        </Typography>
```